tokio/tests/sync_mutex.rs

```diff
@@ -119,15 +119,15 @@ async fn aborted_future_2() {
             let m2 = m1.clone();
             // Try to lock mutex in a future that is aborted prematurely
             timeout(Duration::from_millis(1u64), async move {
-                m2.lock().await;
+                let _g = m2.lock().await;
             })
             .await
             .unwrap_err();
         }
     }
     // This should succeed as there is no lock left for the mutex.
     timeout(Duration::from_millis(1u64), async move {
-        m1.lock().await;
+        let _g = m1.lock().await;
```

Comment:
I think these could also just be 
```suggestion
        let _ = m1.lock().await;
```
since there's nothing else in the scope; we drop the guard immediately in both cases and the binding is only necessary to avoid the must_use warning.